lib/send-mail.ts

```diff
@@ -147,15 +149,25 @@ export async function parseMBoxMessageIDAndReferences(mbox: string):
 export async function sendMail(mail: IParsedMBox,
                                smtpOptions: ISMTPOptions):
     Promise<string> {
+    const transportOpts: SMTPTransport.Options | any = {
+        auth: {
+            pass: smtpOptions.smtpPass,
+            user: smtpOptions.smtpUser,
+        },
+        host: smtpOptions.smtpHost,
+        secure: true,
+    };
+
+    if (smtpOptions.smtpOpts) {
+        const options: any = JSON.parse(smtpOptions.smtpOpts);
+        for (const key of Object.keys(options)) {
+            // transportOpts.key = options[`${key}`];
```

Comment:
Did you want to keep the commented-out line?

Apart from that, I think there is a "more functional" way to do this:

```typescript
Object.entries(JSON.parse(smtpOptions.smtpOpts)).forEach(([key, value]) => transportOpts[key] = value);
```

What do you think?

Reply:
Totally agree.  That is pretty much where I started before running into typing problems (resolved by type any).  Thanks for pointing that out.  Changes have been pushed.